### PR TITLE
Include type declarations in the build

### PR DESCRIPTION
**Pull request by aminya**

### CODE REVIEW

**Comment by aminya** [comment]
@steelbrain We have to write declarations in every project while this project already can include the types 😁

File: package.json [hunk 1]
```diff
@@ -46,6 +48,7 @@
     "rollup": "^2.26.2",
     "rollup-plugin-preserve-shebang": "^1.0.1",
     "semver-compare": "^1.0.0",
+    "shx": "^0.3.3",
```
Comment (steelbrain) [inline package.json:51]: Thoughts on using `rimraf` instead?

https://www.npmjs.com/package/rimraf

Comment (aminya) [reply package.json:51]: No difference. I use shx because it includes other things that I usually need. 

https://github.com/shelljs/shelljs

Comment (aminya) [reply package.json:51]: Is this holding the PR back? I can change to rimraf if you want.

Comment (steelbrain) [reply package.json:51]: I was on a vacation earlier, got back recently

**Review by steelbrain — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (aminya) [reply package.json:51]: Thank you! No problem.

**Pull request merged**